fineract-provider/src/main/resources/META-INF/spring/jndi.xml

```diff
@@ -29,6 +29,12 @@
     	http://www.springframework.org/schema/jee http://www.springframework.org/schema/jee/spring-jee-3.0.xsd">
 
 	<!-- name here must match TestDataSourceConfiguration -->
-	<jee:jndi-lookup jndi-name="java:comp/env/jdbc/mifosplatform-tenants"
-		id="tenantDataSourceJndi" />
+
+    <bean id="tenantDataSourceJndi" 
```

Comment:
It would be clearer to rename this bean name and drop the *Jndi suffix (because not it's not using JNDI anymore...), and according change the use of this bean in the code (search for it...).  Also, it then probably should not be in `jndi.xml` anymore now.